### PR TITLE
Update update-phone-number-and-email-address.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected the navigation path for Editing alternate email, phone, and alternate phone

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3112

File: microsoft-365/admin/manage/update-phone-number-and-email-address.md [hunk 1]
```diff
@@ -35,15 +35,15 @@ This article explains how you, the admin, can change your business phone and ema
   
 If you're looking for how to change your company's profile information, such as company name and address, company phone number, and technical contact information, see [Change your organization's address, technical contact email, and other information](change-address-contact-and-more.md).
   
-## To update your name, phone number, and email address
+## To update your phone number, and email address
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/update-phone-number-and-email-address.md]:
```suggestion
## To update your phone number and email address
```

File: microsoft-365/admin/manage/update-phone-number-and-email-address.md [hunk 2]
```diff
@@ -35,15 +35,15 @@ This article explains how you, the admin, can change your business phone and ema
...
 
-Use the **Settings and Privacy** page to change your name, mobile phone number, and alternate email address. The alternate email address is used for important notifications, such as resetting your admin password (not your computer admin password). 
+Use the **Security Info** page to change your name, mobile phone number, and alternate email address. The alternate email address is used for important notifications, such as resetting your admin password (not your computer admin password).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/update-phone-number-and-email-address.md]:
```suggestion
Use the **Security Info** page to change your mobile phone number and alternate email address. The alternate email address is used for important notifications, such as resetting your admin password (not your computer admin password). 
```

File: microsoft-365/admin/manage/update-phone-number-and-email-address.md [hunk 3]
```diff
@@ -35,15 +35,15 @@ This article explains how you, the admin, can change your business phone and ema
...
+Use the **Security Info** page to change your name, mobile phone number, and alternate email address. The alternate email address is used for important notifications, such as resetting your admin password (not your computer admin password). 
   
 1. Browse to the Microsoft 365 admin center.
 
-2. In the header, select your profile icon \> **My account** \> **Settings and Privacy**.
+2. In the header, select your profile icon \> **My account** \> **Security Info**.
 
-3. In the **Privacy** tab under **Contact Preferences**, select **Contact preferences for admins only**. Update your mobile, phone, and alternate email address. Make sure you use something different from your Microsoft email address for your alternate email address.
+3. In the **Security info** tab Click **Add Method**, select **Phone**, **Alternate Phone**, Or **Email** to add details. To Update your mobile, phone, and alternate email address you can edit them by clicking on **Change**. Make sure you use something different from your Microsoft email address for your alternate email address.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/update-phone-number-and-email-address.md]:
```suggestion
3. In the **Security info** tab, select **Add Method** \> **Phone** \> **Alternate Phone** or **Email** to add details. To update your mobile, phone, and alternate email address details, select **Change**. Make sure you use something different than your Microsoft email address for your alternate email address.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/admin/manage/update-phone-number-and-email-address.md [hunk 4]
```diff
@@ -35,15 +35,15 @@ This article explains how you, the admin, can change your business phone and ema
...
+Use the **Security Info** page to change your mobile phone number and alternate email address. The alternate email address is used for important notifications, such as resetting your admin password (not your computer admin password). 
   
 1. Browse to the Microsoft 365 admin center.
 
-2. In the header, select your profile icon \> **My account** \> **Settings and Privacy**.
+2. In the header, select your profile icon \> **My account** \> **Security Info**.
 
-3. In the **Privacy** tab under **Contact Preferences**, select **Contact preferences for admins only**. Update your mobile, phone, and alternate email address. Make sure you use something different from your Microsoft email address for your alternate email address.
+3. In the **Security info** tab, select **Add Method** \> **Phone** \> **Alternate Phone** or **Email** to add details. To update your mobile, phone, and alternate email address details, select **Change**. Make sure you use something different than your Microsoft email address for your alternate email address.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/update-phone-number-and-email-address.md]:
```suggestion
3. In the **Security info** tab, select **Add Method** \> **Phone** \> **Alternate Phone** or **Email** to add details. To update your mobile, phone, and alternate email address details, select **Change**. Make sure you use something other than your Microsoft email address for your alternate email address.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies Sri, one more please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder please merge this PR. Thanks.

**Pull request merged**